css/horizonAuthenticate.go

```diff
@@ -180,31 +183,49 @@ func (auth *HorizonAuthenticate) authenticateWithExchange(otherOrg string, appKe
 			}
 
 		} else {
-			// Check if the identity is a user, since we know its not an agbot.
+			// Check if the identity is a user, since we know its not an agbot. If an error is returned, check if the identity is a node user.
 			if trace.IsLogging(logger.WARNING) {
 				log.Warning(cssALS(fmt.Sprintf("unable to verify identity %v as agbot, error %v", appKey, err)))
 			}
 			if trace.IsLogging(logger.TRACE) {
 				trace.Debug(cssALS(fmt.Sprintf("attempting authentication request as a user %v", appKey)))
 			}
-			if admin, err := auth.verifyUserIdentity(parts[1], parts[0], appSecret, ExchangeURL()); err != nil {
+			// appkey: {org}/{username} or {org}/iamapikey. 
+			// parts[1] is {username} or iamapikey, parts[0] is {orgId}
+			if admin, username, err := auth.verifyUserIdentity(parts[1], parts[0], appSecret, ExchangeURL()); err != nil {
```

Comment:
I cant add a comment to line 197, but I think that should be a warning now, since we will check for node user next. This is similar to the warning on line 188.